release-notes/WCF-Web-Service-Reference-notes.md

```diff
@@ -4,6 +4,10 @@ WCF Web Service Reference connected services provider is now part of Visual Stud
 
 ## Release History
 
+### 2/7/2018 - Visual Studio 2017 v15.6 Preview 4.0 [27406.0.d15.6]
+* Some improvements in the service metadata download and processing area. 
```

Comment:
~~Some~~ improvements in ~~the~~ service metadata download*ing* and processing ~~area~~.